app/src/main/java/com/example/tasky/feature_agenda/presentation/Agenda.kt

```diff
@@ -202,7 +270,14 @@ fun AgendaContent(
                                         )
                                 },
                                 cardType = agendaItem.cardType,
-                                isChecked = false
+                                isChecked = false,
+                                toggleAgendaCardDropdownVisibility = toggleAgendaCardDropdownVisiblity
+                            )
+                            CardDropdownRoot(
+                                agendaItem = agendaItem,
+                                showAgendaCardDropdown = showAgendaCardDropdown,
+                                toggleAgendaCardDropdownVisibility = toggleAgendaCardDropdownVisiblity,
+                                onAgendaCardActionClick = onAgendaCardActionClick
                             )
```

Comment:
Just putting it here like that will misalign it as on your screenshot. Instead, you need to anchor this dropdown to the composable you want to show it from (the menu icon button on the agenda card). So, in your agenda card, you'd need the following:
```
Box {
    IconButton(...) {
        ...
    }
    CardDropdownRoot(...)
}
```
This will show the drop down above or below the menu button, depending on the available space